haystack/document_stores/es_converter.py

```diff
@@ -185,23 +189,43 @@ def elasticsearch_index_to_document_store(
     es_import.check()
 
     # This import cannot be at the beginning of the file, as this would result in a circular import
-    from haystack.document_stores.elasticsearch import ElasticsearchDocumentStore
+    from haystack.document_stores import ElasticsearchDocumentStore
 
     # Initialize Elasticsearch client
-    es_client = ElasticsearchDocumentStore._init_elastic_client(
-        host=host,
-        port=port,
-        username=username,
-        password=password,
-        api_key=api_key,
-        api_key_id=api_key_id,
-        aws4auth=aws4auth,
-        scheme=scheme,
-        ca_certs=ca_certs,
-        verify_certs=verify_certs,
-        timeout=timeout,
-        use_system_proxy=use_system_proxy,
-    )
+    if ElasticsearchDocumentStore.__module__ == "haystack.document_stores.elasticsearch7":
```

Comment:
This is not a great idea I think, isn't there a better way to know which version are we using?